{"uuid":"a71a24c2-5b5a-42a9-b502-cf0e9594c8f4","historyId":"e2cb8aa82d75498f7b8ac37ab6ad3fa3","status":"broken","statusDetails":{"message":"`cy.visit()` failed trying to load:\n\nhttps://folio-snapshot.dev.folio.org/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer","trace":"CypressError: `cy.visit()` failed trying to load:\n\nhttps://folio-snapshot.dev.folio.org/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158141:82\n    at visitFailedByErr (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:157494:12)\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158140:11\nFrom previous event:\n    at go (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158103:17)\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158169:20\nFrom previous event:\n    at visit (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158167:38)\n    at Context.visit (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158176:14)\nFrom Your Spec Code:\n    at Context.eval (https://folio-snapshot.dev.folio.org/__cypress/tests?p=cypress\\support\\index.js:2689:6)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26)\n  "},"stage":"finished","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"suite","value":"ui-data-import: MARC file upload with the update of instance, holding, and items"}],"links":[],"start":1643375674432,"name":"C343335 MARC file upload with the update of instance, holding, and items","fullName":"\"before all\" hook: navigates to Settings for \"C343335 MARC file upload with the update of instance, holding, and items\"","stop":1643375674435}